### PR TITLE
Add gregurco/GuzzleBundleOAuth2Plugin to readme

### PR DESCRIPTION
**Pull request by gregurco**
- [ ] Bug fix
- [x] New feature
- [ ] BC breaks
- [ ] Deprecations
- [x] Tests pass

Fixed tickets: [#58]
License: MIT

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -177,4 +178,5 @@ This bundle is released under the [MIT license](src/Resources/meta/LICENSE)
 [4]: http://docs.guzzlephp.org/en/latest/request-options.html
 [5]: https://github.com/gregurco/GuzzleBundleWssePlugin
 [6]: https://github.com/gregurco/GuzzleBundleCachePlugin
-[7]: https://github.com/8p/GuzzleBundle/graphs/contributors
+[7]: https://github.com/gregurco/GuzzleBundleOAuth2Plugin
+[8]: https://github.com/8p/GuzzleBundle/graphs/contributors
```
Comment (florianpreusner) [inline README.md]: Can you fix this link also? I know, it's not a modification by you...

Comment (gregurco) [reply README.md]: yep, no problem :slightly_smiling_face:

**Comment by gregurco** [comment]
@florianpreusner fine, could you please merge. Probably something is wrong with VersionEye.

**Comment by florianpreusner** [comment]
Yeah, removed the hooks. Service will be shutdown soon, unfortunately.

**Pull request merged**